tests/helpers/test_service.py

```diff
@@ -457,14 +470,12 @@ def test_async_get_all_descriptions(hass):
         mock_entities["light.kitchen"],
         mock_entities["light.living_room"],
     ]
-    assert (
-        "Not passing an entity ID to a service to target " "all entities is deprecated"
-    ) not in caplog.text
+    assert ("Not passing an entity ID to a service to target "
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4603109)